<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="wave1Label">
     <property name="text">
      <string>Wave1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wave1Area" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="wave1Line">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wave1Button">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="wave2Label">
     <property name="text">
      <string>Wave2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wave2Area" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="wave2Line">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wave2Button">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wave1Button</sender>
   <signal>clicked()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>_wave1Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wave2Button</sender>
   <signal>clicked()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>_wave2Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_wave1Clicked()</slot>
  <slot>_wave2Clicked()</slot>
 </slots>
</ui>
